docs/09-Advanced.md

```diff
@@ -159,6 +159,19 @@ var myPieChart = new Chart(ctx, {
 });
 ```
 
+In addition to all tooltip properties, a `dataPoints` property is added containing an `array[CustomTooltipDataPoint]`.
+
+#### CustomTooltipDataPoint
```

Comment:
Sorry for the delay, I move this documentation from Advanced to proper Chart Configuration part.